### DISCUSSION TITLE
Helm release promotion with GitLab

### DISCUSSION THREAD

**Discussion by luckielordie** [fluxcd/flux2#3633] [Q&A] [answered]
Hi Flux Community! 

Our team has been investigating using FluxCD for our deployment process, we really like the approach with our test cluster. However when we look forward in our plan to promoting our releases out through our environments. 

The [Promote Flux Helm Release with Github Actions](https://fluxcd.io/flux/use-cases/gh-actions-helm-promotion/#define-the-promotion-github-workflow) page has been pretty informative as a strategy that we will try to emulate. But I'm looking for any advice or gotchas for configuring this system in a GitLab environment. Has anyone else attempted this kind of deployment flow?

Many Thanks.

**Comment by stefanprodan** [comment, marked as answer]
Flux notification-controller doesn't support triggering GitLab CI. Please open an issue on GitLab.com, I'm sure they would want this feature for their Flux integration. You could also ping [Viktor Nagy](https://github.com/nagyv) ref https://github.com/fluxcd/flux2/discussions/3595#discussion-4862181
